scripts/build.sh

```diff
@@ -390,41 +439,104 @@ clean_previous_packages() {
 	rm ${HASH_DIR}/${1}.pkglist
 }
 
-build_dpkg() {
-	if [ -e "${DPKG_OVERLAY}/packages/Packages.gz" ] ; then
-		chroot ${DPKG_OVERLAY} apt update || exit_err "Failed apt update"
+mk_kernoverlay() {
+	# Generate kernel overlay (but not mount).
+	# This makes our debian directory and kernel config used for building
+	# debian folder is required to install pre-build dependencies.
+	mkdir ${KERNTMP}
+	mkdir ${KERNWRK}
+	stored_cwd=$(pwd)
+	cp -r ${SOURCES}/kernel/* ${KERNTMP} || exit_err "Failed to copy sources"
+	apt install -y flex bison dwarves libssl-dev > /dev/null
```

Comment:
Is it possible that we'll be building other kernel modules later that need different packages installed? Perhaps this should be in the JSON manifest instead for each repo? Like "kernel_pkg_depends": or something?

Reply:
I'm fine going without this, we can add that functionality later when needed.